package.json

```diff
@@ -688,6 +688,10 @@
           "default": false,
           "description": "Falls back to the legacy (lightweight) ReadLine experience. This will disable the use of PSReadLine in the PowerShell Integrated Console."
         },
+        "powershell.integratedConsole.useTrueClear": {
```

Comment:
How about something like `powershell.integrateConsole.clearScrollbackBuffer`. Or slightly more futureproof maybe: `"powershell.integratedConsole.clearAction": "scrollback" | "clear"`. Something with the term `scrollback` in it, just since `TrueClear` sounds a bit like an background check or a bad liquor to me.

Reply:
LOL

Ok. I kinda like the clearAction one. Let's go with that

Reply:
went with `forceClearScrollbackBuffer` because we can't change the behavior of clear-host other than forcing a full clear.